src/app/core/services/loader.service.spec.ts

```diff
@@ -29,7 +29,9 @@ describe('LoaderService', () => {
     const spinner = document.getElementById('spinner');
     const service: LoaderService = TestBed.get(LoaderService);
     service.hideLoader();
-    expect(spinner.style.display).toEqual('none');
+    setTimeout(() => {
+      expect(spinner.style.display).toEqual('none');
+    }, 200);
```

Comment:
It's better to use `fakeAsync` and `tick` instead of using setTimeout in the tests.